site/src/components/components/ComponentsList.module.css

```diff
@@ -61,6 +61,10 @@
   --saltIcon-color: var(--salt-status-success-foreground);
 }
 
+.status.lab svg {
```

Comment:
Not a blocker but this could be combined with the styling above

Reply:
Good spot. Fixed it.